src/events/AudioEvents.hxx

```diff
@@ -51,6 +133,83 @@ struct AudioPlay3DEvent
   Coordinate3D position;
 };
 
+/**
+ * @brief event playing a track with reverb effect
+ */
+struct AudioPlayReverbEvent
+{
+  SoundtrackID ID;
+  StandardReverbProperties reverb_properties;	
+};
+
+/**
+ * @brief event playing a track with echo effect
+ */
+struct AudioPlayEchoEvent
+{
+  SoundtrackID ID;
+  EchoProperties echo_properties;	
+};
+
+
+/**
+ * @brief event triggering a track with reverb
+ */
+struct AudioTriggerReverbEvent
+{
+  AudioTrigger trigger;
+  StandardReverbProperties reverb_properties;
+};
+
+/**
+ * @brief event triggering a track with echo
+ */
+struct AudioTriggerEchoEvent
+{
+  AudioTrigger trigger;
+  EchoProperties echo_properties;
+};
+
+/**
+ * @brief event playing a track with reverb effect with 3D sound
+ */
+struct AudioPlayReverb3DEvent
+{
+  SoundtrackID ID;
+  Coordinate3D position;
+  StandardReverbProperties reverb_properties;	
+};
+
+/**
+ * @brief event playing a track with echo effect with 3D sound
+ */
+struct AudioPlayEcho3DEvent
+{
+  SoundtrackID ID;
+  Coordinate3D position;
+  EchoProperties echo_properties;	
```

Comment:
consider adding a doxygen comment to each struct member  for documentation.

Like 
```cpp
  SoundtrackID ID; /// the ID of the sountrack, described in the json file
  Coordinate3D position; /// the position that will be caluclated by function xxx or set in function xxx.
```